packages/bits/spec/components/menu/menu.e2e.ts

```diff
@@ -30,82 +23,7 @@ describe("USERCONTROL Menu", () => {
         await menu.toggleMenu();
     });
 
-    describe("> basic menu", () => {
-        it("should show menu", async () => {
-            expect(await menu.isMenuOpened()).toBe(false);
-            await menu.toggleMenu();
-            expect(await menu.isMenuOpened()).toBe(true);
-        });
-
-        it("should return menu titles", async () => {
-            await menu.toggleMenu();
-            expect(await menu.getItemTextArray())
-                .toEqual([
-                    "Menu Item",
-                    "Selected menu item",
-                    "Switched checked",
-                    "Switch unchecked",
-                    "Disabled switch",
-                    "Disabled action",
-                    "Disabled link",
-                    "Disabled option",
-                    "Menu item with checkbox",
-                    "Menu item with icon",
-                    "Link menu item",
-                    "Export PDF",
-                    "Menu Item1",
-                    "Menu Item2",
-                    "Menu Item3",
-                ]);
-        });
-
-        it("should have enabled items by default", async () => {
-            await menu.toggleMenu();
-            expect(await menu.getMenuItemByContainingText("Export PDF").isDisabledItem()).toBe(false);
-            expect(await menu.getMenuItemByContainingText("Switched checked").isDisabledItem()).toBe(false);
-            expect(await menu.getMenuItemByContainingText("Menu item with checkbox").isDisabledItem()).toBe(false);
-            expect(await menu.getMenuItemByContainingText("Link menu item").isDisabledItem()).toBe(false);
-        });
-    });
-
-    describe("> disabled items", () => {
-        beforeEach(async () => {
-            await menu.toggleMenu();
-        });
-        it("should have disabled items", async () => {
-            expect(await menu.getMenuItemByContainingText("Disabled switch").isDisabledItem()).toBe(true);
-            expect(await menu.getMenuItemByContainingText("Disabled action").isDisabledItem()).toBe(true);
-            expect(await menu.getMenuItemByContainingText("Disabled link").isDisabledItem()).toBe(true);
-            expect(await menu.getMenuItemByContainingText("Disabled option").isDisabledItem()).toBe(true);
-        });
-        it("should locate disabled checkbox", async () => {
-            const disabledOptionItem = menu.getMenuItemByContainingText("Disabled option");
-            const checkbox: CheckboxAtom = Atom.findIn(CheckboxAtom, disabledOptionItem.getElement());
-            expect(await checkbox.isChecked()).toBe(false);
-        });
-        it("should locate disabled switch", async () => {
-            const disabledSwitchItem = menu.getMenuItemByContainingText("Disabled switch");
-            const switchAtom = Atom.findIn(SwitchAtom, disabledSwitchItem.getElement());
-            expect(await switchAtom.isOn()).toBe(false);
-        });
-    });
```

Comment:
It seems like there are no such cases in visual tests

Reply:
Discussed offline - _Disabled switch_, _Disabled action_, _Disabled link_ and _Disabled option_ are verified in another visual tests.